src/tunacode/ui/app.py

```
@@ -397,6 +408,33 @@ def _update_resource_bar(self) -> None:
+                self._stream_active = True
+                self.streaming_output.add_class("active")
+                self.loading_indicator.remove_class("active")
+
+            now = time.monotonic()
+            elapsed_ms = (now - self._last_stream_update) * 1000
+            if elapsed_ms >= self.STREAM_THROTTLE_MS or is_first_chunk:
+                self._last_stream_update = now
+                self.streaming_output.update("".join(self._stream_buffer))
+        except Exception:
+            # Silently drop UI errors to avoid breaking agent loop
+            pass
```

Ruff (0.15.0)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)